### DISCUSSION TITLE
rust-course/3.5.2自引用结构体中

### DISCUSSION THREAD

**Discussion by swellingWu** [sunface/rust-course#511] [内容建议]
struct RefWithMe<'a> {
    value: String,
    pointer_to_value: &'a str,
}
fn main(){
    let s = "aaa".to_string();
    let v = SelfRef {
        value: s,
        pointer_to_value: &s
    };
}

——>main函数中 SelfRef 是书写错误么，还是象征 引用结构体 的意思

**Comment by sunface** [comment]
书写错误。。感谢提醒，已修复
